org.svxlink.Qtel.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.svxlink.Qtel
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+command: qtel
+rename-desktop-file: qtel.desktop
+rename-icon: qtel
+finish-args:
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/doc
+  - /share/man
+  - /man
+  - "*.a"
+  - "*.la"
+
+modules:
+  - name: mm-common
+    cleanup:
+      - /bin
+      - /lib
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/mm-common/1.0/mm-common-1.0.6.tar.xz
+        sha256: b55c46037dbcdabc5cee3b389ea11cc3910adb68ebe883e9477847aa660862e7
+
+  - name: libsigc++-2.0
+    sources:
+      - type: archive
+        url: https://github.com/libsigcplusplus/libsigcplusplus/releases/download/2.12.1/libsigc++-2.12.1.tar.xz
+        sha256: a9dbee323351d109b7aee074a9cb89ca3e7bcf8ad8edef1851f4cf359bd50843
+
+  - name: libpopt
+    sources:
+      - type: archive
+        url: https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
+        sha256: c25a4838fc8e4c1c8aacb8bd620edb3084a3d63bf8987fdad3ca2758c63240f9
+
+  - name: libjsoncpp
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.tar.gz
+        sha256: f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2
+
+  - name: tcl
+    subdir: unix
+    build-options:
+      no-debuginfo: true
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.13/tcl8.6.13-src.tar.gz
+        sha256: 43a1fae7412f61ff11de2cfd05d28cfc3a73762f354a417c62370a54e2caf066
+
+  - name: libgsm
+    no-autogen: true
+    build-options:
+      make-install-args:
+        - INSTALL_ROOT=/app
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: http://deb.debian.org/debian/pool/main/libg/libgsm/libgsm_1.0.22.orig.tar.gz
+        sha256: f0072e91f6bb85a878b2f6dbf4a0b7c850c4deb8049d554c65340b3bf69df0ac
+      - type: patch
+        path: libgsm-makefile.patch
+
+  - name: qtel
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DDO_INSTALL_CHOWN=No
+    subdir: src
+    sources:
+      - type: git
+        url: https://github.com/sm0svx/svxlink.git
+        commit: a4e9aec8762d26236e736b0518026455a062dc63
```

Comment:
I am using latest Git commit since the last stable version was released in 2021 and is outdated now. Git version from master contains many fixes and improvements including AppStream metadata and desktop fixes needed for this Flatpak.

Reply:
https://github.com/sm0svx/svxlink/issues/662